installer/mac/README.md

```diff
@@ -40,11 +40,22 @@ The Xcode Command Line Tools package includes utilities for compiling code from
     rm /tmp/Miniconda3-latest-MacOSX-x86_64.sh
     ```
 
+## Download HNN source code
+
+- The following commands will download the hnn source code and compile HNN's mod files for NEURON. We use the directory `hnn_source_code` for consistency with all of our instructions, but any directory can be used. You can use `git` if you prefer.
+
+    ```bash
+    curl -OL https://github.com/jonescompneurolab/hnn/releases/latest/download/hnn.tar.gz
+    mkdir hnn_source_code
+    tar -x --strip-components 1 -f hnn.tar.gz -C hnn_source_code
```

Comment:
Note that we're now having users download the source code **before** attempting to call `environment.yml` (bug).